index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Mobile Skeleton</title>
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@700&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <header class="logo-sec">
+      <a href="#" class="wel">Welcome</a>
+      <div class="menu">
```

Comment:
- [ ] Let's change this `div` tag for a `nav` tag. This is good for semantics and accessibility purposes. It makes your code more legible for both humans and machines. [You can read more here.](https://www.freecodecamp.org/news/semantic-html5-elements/)